js-intro.js

```diff
@@ -11,51 +11,118 @@ var temp1 = 35
 var temp2 = 350
 var temp3 = 212
 
-
-
+// Boiling point = 212
+const boilingTemp = (temp) => {
+    // Checks to see if input is of number data type
+    if (typeof temp === "number") {
+        if (temp === 212) {
+            return `${temp} is at boiling point.`;
+        } else if (temp < 212) {
+            return `${temp} is below boiling point.`;
+        } else if (temp > 212) {
+            return `${temp} is above boiling point.`;
+        }
+      // If not of number data type, message appears         
+    } else {
+        return 'Please enter a number and try again.';
+    }
+}
 
 // --------------------2) Create a function that multiplies each number in the array by 5 using a for loop.
 // Use the test variable provided below. Expected outcome: [15, 35, 0, 30, -45]
 
-var myNumbers1 = [3, 7, 0, 6, -9]
-
+var myNumbers1 = [3, 7, 0, 6, -9];
 
+// Function to run array items * 5 using for loop
+const timesFive = (arr) => {
+    // Declare empty array to store calculated results
+    let calculatedArray = [];
+    // for loop to stop running once variable i is one less than the length of the inputed array
+    for (var i = 0; i < arr.length; i++) {
+        calculatedArray.push(arr[i] * 5);
+    }
+    return calculatedArray;
+}
 
 
 // --------------------3) Create a function that multiplies each number in the array by 5 using map.
 // Use the test variable provided below. Expected outcome: [40, -35, 0, 30, 10]
 
 var myNumbers2 = [8, -7, 0, 6, 2]
 
-
-
+// Function to run array item * 5 using map
+const timesFiveV2 = (arr) => {
+    let calculatedArray = [];
+    calculatedArray = arr.map(el => el * 5);
+    return calculatedArray;
```

Comment:
You can save yourself a couple lines of code by returning your mapped array outright, rather than creating a new variable in order to turn around and immediately return it. Map is an accessor which doesn't permanently change to original array, so if you wanted to use those values later in your code you would need to save it to a new variable like you did. But for this purpose, you can save yourself some key strokes :) Also map returns an array on it's own so the empty array isn't necessary